### DISCUSSION TITLE
Low battery notification

### DISCUSSION THREAD

**Discussion by evlo** [Koenkk/zigbee2mqtt#16058] [Devices]
How can I easily get low battery notification on my email or something for any device that is running out of battery in Z2M?

What happens right now is I do not check dashboard for some time and then device is missing, just nowhere to be found in z2m and after some time I figure that it did run out of battery (also how can I make device not disappear when it runs out of battery)?

**Comment by evlo** [comment]
Bumping this up as I still wonder what is the best practice to set this up.

**Comment by lux73** [comment]
afaik only with using your preferred HomeAutomation Software
